src/views/alpha/FeedCard.tsx

```diff
@@ -264,19 +272,24 @@ export const ProfileHandle = ({ user, shorten = false }: { user: User; shorten?:
     }
   }, []);
 
+  console.log('user', user);
+
   return (
     <Popover
       isShowOnHover={true}
+      placement={props.popoverPlacement}
       content={
-        <ProfilePreview
-          address={user.address}
-          context={{
-            data: getProfilePreivewDataFromUser(user),
-            loading: false,
-            refetch: () => {},
-            fetchMore: () => {},
-          }}
-        />
+        <div className="w-80">
+          <ProfilePreview
+            address={user.address}
+            context={{
+              data: getProfilePreivewDataFromUser(user),
```

Comment:
NB: Instead of generic utils I think modules/profiles is a good spot for this function. convertToProfile Also instead of converting in the caller consider accepting User and running the conversion within the profile component. Maybe even drop if this is the only spot needing the conversion.

Reply:
@austinmilt for your consideration

Reply:
> instead of converting in the caller consider accepting User and running the conversion within the profile component

IMO we should be removing Graph QL types from our components completely and convert to the data type needed by the component before passing data through. This helps keep components "dumb" and facilitates refactoring.